### PR TITLE
P-899 Pallet Balances Related: EVM Decimal Migration (Code view only)

### PR DESCRIPTION
**Pull request by wangminqi**
**Do not merge this PR.**
The PR create for code view purpose only.
The official PR is on p-895

### CODE REVIEW

**Comment by wangminqi** [comment]
Except ts-test (which is disabled), this branch is ready.

File: runtime/litentry/src/xcm_config.rs [hunk 1]
```diff
@@ -174,18 +174,19 @@ parameter_types! {
 	/// Xcm fees will go to the treasury account
 	pub XcmFeesAccount: AccountId = Treasury::account_id();
 	pub const MaxAssetsIntoHolding: u32 = 64;
+	pub const WeighToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```
Comment (silva-fj) [inline runtime/litentry/src/xcm_config.rs]:
```suggestion
	pub const WeightToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```

File: runtime/litmus/src/lib.rs [hunk 2]
```diff
@@ -419,7 +421,8 @@ impl pallet_utility::Config for Runtime {
 }
 
 parameter_types! {
-	pub const TransactionByteFee: Balance = MILLICENTS / 10;
+	pub const TransactionByteFee: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
+	pub const WeighToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```
Comment (silva-fj) [inline runtime/litmus/src/lib.rs]:
```suggestion
	pub const WeightToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```

File: runtime/litmus/src/xcm_config.rs [hunk 3]
```diff
@@ -173,18 +173,19 @@ parameter_types! {
 	/// Xcm fees will go to the treasury account
 	pub XcmFeesAccount: AccountId = Treasury::account_id();
 	pub const MaxAssetsIntoHolding: u32 = 64;
+	pub const WeighToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```
Comment (silva-fj) [inline runtime/litmus/src/xcm_config.rs]:
```suggestion
	pub const WeightToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```

File: runtime/rococo/src/lib.rs [hunk 4]
```diff
@@ -515,15 +520,16 @@ impl pallet_utility::Config for Runtime {
 }
 
 parameter_types! {
-	pub const TransactionByteFee: Balance = MILLICENTS / 10;
+	pub const TransactionByteFee: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
+	pub const WeighToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```
Comment (silva-fj) [inline runtime/rococo/src/lib.rs]:
```suggestion
	pub const WeightToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```

File: runtime/rococo/src/xcm_config.rs [hunk 5]
```diff
@@ -173,18 +173,19 @@ parameter_types! {
 	/// Xcm fees will go to the treasury account
 	pub XcmFeesAccount: AccountId = Treasury::account_id();
 	pub const MaxAssetsIntoHolding: u32 = 64;
+	pub const WeighToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```
Comment (silva-fj) [inline runtime/rococo/src/xcm_config.rs]:
```suggestion
	pub const WeightToFeeFactor: Balance = WEIGHT_TO_FEE_FACTOR; // 10^6
```

**Review by silva-fj — APPROVED** [review]
LGTM

**Comment by wangminqi** [comment]
This PR has been merged into branch P-895. 
This branch will not be maintained.

**Pull request closed without merging**